RobotCrawler/src/main/java/parser/empik/EmpikUrlResolver.java

```diff
@@ -0,0 +1,18 @@
+package parser.empik;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+class EmpikUrlResolver {
+
+    static String resolveUrl(String url) {
+        URI empikUri = null;
+        try {
+            empikUri = new URI("http://empik.com/");
+        } catch (URISyntaxException e) {
+            e.printStackTrace();
+        }
+        return empikUri.resolve(url).toString();
```

Comment:
I've improved that in this commit: https://github.com/team-javapocalypse-ja/Robot/pull/6/commits/4de45ff2b508f0fe7665e9da1988dc13c10e78e8